docs/architecture/transactions/kernel.md

```diff
@@ -90,19 +134,22 @@ The transaction script can be used to authenticate the transaction by increasing
     end
 ```
 
-_Note: The executing account must expose the `auth_tx_rpo_falcon512` function in order for the transaction script to call it._
+!!! note
+    - The executing account must expose the `auth_tx_rpo_falcon512` function in order for the transaction script to call it.
 
+## Epilogue
 
-## The Epilogue
-The Epilogue finalizes the transaction. It
+The Epilogue finalizes the transaction. It does the following:
```

Comment:
nit: can you lower case Epilogue

Reply:
just do it these in future directly and i'll accept

Reply:
done, these are easy to miss ..